public/docs/_examples/displaying-data/dart/web/main.dart

```diff
@@ -0,0 +1,16 @@
+// #docregion
+import 'package:angular2/bootstrap.dart';
+import 'package:displaying_data/app_ctor_component.dart';
+import 'package:displaying_data/app_component_1.dart' as v1;
+import 'package:displaying_data/app_component_2.dart' as v2;
+import 'package:displaying_data/app_component_3.dart' as v3;
+import 'package:displaying_data/app_component_final.dart' as v4;
+
+main() {
+//  bootstrap(v1.AppComponent);
+//  bootstrap(v2.AppComponent);
+//  bootstrap(v3.AppComponent);
+  bootstrap(v4.AppComponent);
+  bootstrap(AppCtorComponent);
```

Comment:
Sorry, I should've been more specific. v1, v2, etc. seemed fine to me, since v is a very common abbreviation.

It's Ctor that we don't usually see in Dart. Maybe we should change it to "Creator" or something other than "Constructor"? I don't want it confused with a Dart constructor.